android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/social/wechat/WechatSocialSession.kt

```diff
@@ -0,0 +1,56 @@
+package io.logto.sdk.android.auth.social.wechat
+
+import android.app.Activity
+import android.net.Uri
+import com.tencent.mm.opensdk.constants.ConstantsAPI
+import com.tencent.mm.opensdk.modelbase.BaseResp
+import com.tencent.mm.opensdk.modelmsg.SendAuth
+import com.tencent.mm.opensdk.openapi.WXAPIFactory
+import io.logto.sdk.android.auth.social.SocialSession
+import io.logto.sdk.android.completion.Completion
+import io.logto.sdk.android.exception.LogtoException
+import io.logto.sdk.core.util.GenerateUtils
+
+class WechatSocialSession(
+    override val context: Activity,
+    override val redirectTo: String,
+    override val callbackUri: String,
+    override val completion: Completion<String>,
+) : SocialSession {
+    companion object {
+        const val APP_ID = "app_id"
```

Comment:
could this be defined in the SocialSession interface? looks like the same for all the social sessions?

Reply:
Shall we define a static value inside an interface?

Reply:
I think it's good to keep it separated, for it may be "client_id" in some cases.

Reply:
Updated.
Remove this already, use "app_id" directly.